vpr/src/util/ndmatrix_serdes.h

```diff
@@ -0,0 +1,61 @@
+#ifndef NDMATRIX_SERDES_H_
+#define NDMATRIX_SERDES_H_
+
+#include <functional>
+#include "vtr_ndmatrix.h"
+#include "vpr_error.h"
+#include "matrix.capnp.h"
```

Comment:
Similarly, since this is for vtr::NdMatrix, perhaps this should be with libvtrutil?

Although it would then mean linking capnproto with libvtrutil which may be too heavy weight. Should think about which is best.

Reply:
I'd be open to moving it to something akin to libvtrutil, but agree that the capnproto dependency makes that unpalatable.